openedx/core/djangoapps/theming/storage.py

```diff
@@ -17,30 +23,22 @@ class ComprehensiveThemingAwareMixin(object):
     """
     def __init__(self, *args, **kwargs):
         super(ComprehensiveThemingAwareMixin, self).__init__(*args, **kwargs)
-        COMP_THEME_DIR = getattr(settings, "COMP_THEME_DIR", "")  # pylint: disable=invalid-name
```

Comment:
Why `getattr` instead of accessing this directly?

Since we define that value by default, the only time it should ever
be undefined means that something pretty horrible has gone wrong.

In such an event, that this code would break is actually a feature, not a bug.